packages/expo-notifications/src/registerTaskAsync.ts

```diff
@@ -3,8 +3,9 @@ import { UnavailabilityError } from 'expo-modules-core';
 import BackgroundNotificationTasksModule from './BackgroundNotificationTasksModule';
 
 /**
- * Call `registerTaskAsync` to set a callback (task) that will run in response to when a notification is received while the app is in foreground, background, or terminated.
- * When app is terminated, only a [data message](https://firebase.google.com/docs/cloud-messaging/concept-options#data_messages) (Android) / [background notification](https://developer.apple.com/documentation/usernotifications/pushing-background-updates-to-your-app#Create-a-background-notification) (iOS) triggers the task execution.
+ * Call `registerTaskAsync` to set a callback (task) that runs when a notification is received while the app is in foreground, background, or terminated.
+ * Only on Android, the task also runs in response to a notification action press when the app is backgrounded or terminated.
+ * When app is terminated, only a [Headless Background Notification](https://docs.expo.dev/push-notifications/what-you-need-to-know/#headless-background-notifications) triggers the task execution.
```

Comment:
```suggestion
 * When the app is terminated, only a [Headless Background Notification](/push-notifications/what-you-need-to-know/#headless-background-notifications) triggers the task execution.
```

We don't need to use absolute URLs internally.